### PR TITLE
S3plugin

### PR DESCRIPTION
**Pull request by marcocast**
Moved scheduler-api dependency to compileOnly in order to avoid vfs-s3 dependency duplication problem in the release

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -1,2 +1,2 @@
 #Fri, 19 Aug 2016 12:03:02 +0200
-projectVersion=7.16.0-SNAPSHOT
+projectVersion=7.18.0-SNAPSHOT
```
Comment (fviale) [inline gradle.properties:2]: I don't think this PR should make a change to the snapshot version, should it ?

Comment (marcocast) [reply gradle.properties:2]: yes it should not. that is an outdated diff...i don t know why you can see it :)

**Comment by fviale** [comment]
:+1:

**Comment by tobwiens** [comment]
👍

**Pull request merged**